test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ServiceBusEndToEndTests.cs

```diff
@@ -296,6 +296,12 @@ private async Task ServiceBusEndToEndInternal(IHost host = null)
                     "      \"MaxAutoRenewDuration\": \"00:05:00\",",
                     "      \"MaxConcurrentCalls\": 16",
                     "  }",
+                    "  \"SessionHandlerOptions\": {",
```

Comment:
You should add a full E2E test that exercises the new session support.

Reply:
test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ServiceBusSessionsEndToEndTests.cs